src/shared/lib/oql/oqlfilter.d.ts

```diff
@@ -1,83 +0,0 @@
-//export function filterCBioPortalWebServiceData(oql_query, data, opt_default_oql, opt_by_oql_line, opt_mark_oql_regulation_direction) {
```

Comment:
@adamabeshouse why can this be removed now?

Reply:
we no longer need a type declaration file because the actual file is now in typescript. before, this served as like a typescript interface.